### ISSUE TITLE
[QUESTION] Does it work in mono repos

### ISSUE THREAD

**Issue by lvegerano** [suchitadoshi1987/related-files-hopper#14]
I have a mono repo that is structure more or less like:
```
my-app
  |_ api
  |_ web (ember app)
    |_ app
```
I have components, in-repo addons and we're in the middle of octane migration (co-located components and classic)

I can't get this to work and I wonder if is my config or a config limitation.

**Comment by suchitadoshi1987** [comment]
@lvegerano yes, it should work. Is there a repo where I can repro the issue you are facing?

**Comment by MichalBryxi** [comment]
My very wild guess will be that you might have hit exactly the same issue I did: #15

**Comment by lvegerano** [comment]
@suchitadoshi1987 I do not have one but I can try and put something together later today

**Comment by suchitadoshi1987** [comment]
@lvegerano is your repo using pods too?

**Comment by lvegerano** [comment]
@suchitadoshi1987 sorry I haven't had a chance to create a repro repo. But no Im using classic structure.
